ledger-service/jwt/src/main/scala/com/digitalasset/jwt/KeyUtils.scala

```diff
@@ -42,6 +42,21 @@ object KeyUtils {
     }
   }
 
+  /**
+    * Reads an EC public key from a X509 encoded file.
+    * These usually have the .crt file extension.
+    */
+  def readECPublicKeyFromCrt(file: File): Try[ECPublicKey] = {
```

Comment:
This function is identical to the above `readRSAPublicKeyFromCrt`, except for the final type cast. Could the common implementation be factored out?

Reply:
Yes i tried to do this but got stuck in handling the Try wrapped in \/ from the calling code.  I will do a removal of some duplication as a follow-up PR, if that sounds okay?